qa/tasks/cephfs/filesystem.py

```diff
@@ -615,9 +615,13 @@ def required_client_features(self, *args, **kwargs):
         c = ["fs", "required_client_features", self.name, *args]
         return self.mon_manager.run_cluster_cmd(args=c, **kwargs)
 
-    # In Octopus+, the PG count can be omitted to use the default. We keep the
-    # hard-coded value for deployments of Mimic/Nautilus.
-    pgs_per_fs_pool = 8
```

Comment:
some tests use this:

````
2021-05-12T04:58:07.911 INFO:tasks.cephfs_test_runner:======================================================================
2021-05-12T04:58:07.911 INFO:tasks.cephfs_test_runner:ERROR: test_add_data_pool_application_metadata (tasks.cephfs.test_admin.TestAdminCommands)
2021-05-12T04:58:07.912 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2021-05-12T04:58:07.912 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2021-05-12T04:58:07.912 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_2905d0548ee49f87d0062b4d26b8e1d9100d248d/qa/tasks/cephfs/test_admin.py", line 87, in test_add_data_pool_application_metadata
2021-05-12T04:58:07.912 INFO:tasks.cephfs_test_runner:    mon_cmd('osd', 'pool', 'create', pool_name, str(self.fs.pgs_per_fs_pool))
2021-05-12T04:58:07.913 INFO:tasks.cephfs_test_runner:AttributeError: 'Filesystem' object has no attribute 'pgs_per_fs_pool'
````

From: /ceph/teuthology-archive/pdonnell-2021-05-12_04:01:31-fs-wip-pdonnell-testing-20210511.232042-distro-basic-smithi/6110701/teuthology.log

Reply:
test_misc.py too

Reply:
and test_data_scan

Reply:
I will fix it. For the data scan one I will add one `fs.get_pool_pg_num()` helper. For the others I will switch to pg autoscale mode.